src/main/java/net/revelc/code/formatter/jsoup/JsoupBasedFormatter.java

```diff
@@ -70,6 +75,30 @@ public String doFormat(final String code, final LineEnding ending) {
         // but our tests fail to do so thus we are duplicating this until jsoup fixes bug.
         formattedCode = REMOVE_TRAILING_PATTERN.matcher(formattedCode).replaceAll("");
 
+        // TODO: Fixing jsoup counting issue which occurs when more than one indentation (jsoup can have mixed line
```

Comment:
I may need to document here more to be clear on what this is doing.

- If indent amount > 1 as only > 1 has the bug
- setup the current lineSize counter
- setup the new lineSize counter
- setup remainder
- Jsoup does not adhere to all same line endings.  So \r? as that happens on windows only in some areas based on their results.  So that along with \n or not.
- The resulting array is the entire thing that jsoup had just formatted
- Create an array to add elements into as we process the fix from the array
- loop through all lines of the array
- If the lineSize is 0, there is nothing to do as no indention on that line.
- Get the newLineSize by trimming all leading spaces
- If line size of original does not match that of new one, attempt to get the remainder by subtracting and dividing the result.
- If remainder is > 0, they are wrong on the count so just substring it properly
- In all cases in that loop either untouched or touched are added to the new array list
- Once completed, using String.join with the line endings we actually want against all lines.

The result here is that the first commit issue where they have many lines counted wrong, those are all corrected.